www/docs/adapters/dynamodb.md

```diff
@@ -21,7 +21,7 @@ npm install next-auth @next-auth/dynamodb-adapter@canary
 
 2. Add this adapter to your `pages/api/auth/[...nextauth].js` next-auth configuration object.
 
-You need to pass `aws-sdk` to the adapter in addition to the table name.
+You need to pass `DocumentClient` from `aws-sdk` to the adapter in addition to the table name.
```

Comment:
Maybe say that you need to pass a `DocumentClient` instance, see you construct it with the `new` keyword?